helm-chart/dash0-operator/README.md

```diff
@@ -2050,15 +2093,6 @@ spec:
         - /webhooks/0/clientConfig/caBundle
 ```
 
-## Notes on Azure AKS
-
-In [AKS](https://azure.microsoft.com/products/kubernetes-service) clusters that have the
-[Azure Policy add-on](https://learn.microsoft.com/azure/aks/use-azure-policy) enabled, it is highly recommended to
-[use a volume for filelog offsets](#providing-a-filelog-offset-volume)
-instead of the default filelog offset config map.
-Using the default config map filelog offset storage in AKS clusters with this add-on can lead to severe performance
-issues.
-
```

Comment:
This section did not actually change, I only moved it up so all cloud-vendor specific things are in one place.